### ISSUE TITLE
[EN][Talent][Paladin] Acts of Sacrifice

### ISSUE THREAD

**Issue by Wingnanx** [Atlantiss/BugTracker#7439]
The talent 'Acts of Sacrifice' is not working properly:
http://cata.openwow.com/spell=85446

The mana/cooldown reduce part of the talent works fine, the second part is saying the following: 
'In addition, your Cleanse will remove one movement impairing effect if cast on yourself.' 

Although not stated here, there is a dispell priority along with this. This allows dispelling root->slow, 

Example: you dispel cone of cold root (talented) before cone of cold slow, this allows you to close the gap between you and the mage when you cleansed the root, while waiting for gcd to cleanse the slow effect afterwards. Currently on Atlantiss you dispell the oldest debuff and in case of cone of cold always the slow before the root.

As an example check the following video's from vanguards on retail s9/s10 and s11.

**Season 9:**
https://youtu.be/mbbE0aRNU0M?t=2m17s (around 2:22) dispel frost nova root before chains of ice slow.

https://youtu.be/yz1xgp0dR24?t=5m9s (around 5:14) dispel frost nova root before cone of cold slow.

https://youtu.be/p8BNqLzN97E?t=2m47s (around 2:52) dispel frost nova root before frostbolt slow.

**Season 10:**
https://youtu.be/Kaa-4wIGelw?t=5m48s (around 5:50) dispel frost nova root before frost armor slow.

https://youtu.be/Kaa-4wIGelw?t=8m36s (around 8:40) dispel cone of cold root before cone of cold slow before frost armor slow.

https://youtu.be/Kaa-4wIGelw?t=9m7s (around 9:10) dispel cone of cold root before cone of cold slow.

**Season 11:**
https://youtu.be/OxIuJZoDIlw?t=17m25s (happens at 17:28) old debuff cone of cold slow, new debuff frost nova -> dispell frost nova before cone of cold.

https://youtu.be/b2QADmYk594?t=2m41s (happens around 2:43) dispell cone of cold root before cone of cold slow.

To sum things up, you need to dispel root before slow, current situation is you dispel oldest debuff from yourself which is wrong! This is gamebreaking while playing retribution in PvP against mages etc.

**Comment by karolnpc** [comment]
Bump

**Comment by karolnpc** [comment]
Bump

**Comment by karolnpc** [comment]
Bump

**Comment by karolnpc** [comment]
Bump

**Comment by karolnpc** [comment]
Bump

**Comment by calladin** [comment]
BUMP #Jenova pls help ^^

**Comment by karolnpc** [comment]
Can you fix this please? It's really annoying.

**Comment by lamaATS** [comment]
Fixed in rev. 5207

**Comment by blinkmaster** [comment]
https://www.youtube.com/watch?v=OxIuJZoDIlw&feature=youtu.be&t=17m25s
according to this link provided, in 7 mins 45 - 49 secs you can see pala dispels slow first even thou he have root on him. Imo this is random chance not like it should 100% target roots.

**Comment by lamaATS** [comment]
In this video at 7:48 I see Cone of Cold and Improved Cone of Cold applied at 7:47 and then first dispel at 7:48 and second at the same time. Problem here is that when first dispel occurred time to expire was invisible because buffs/debuffs blinking and both was dispelled at the same time probably first by Paladin and the seconds by Priest or in revered order so for me it's not enough sorry.
If You see time to expire both Cone of Cold effects give me a screen shot from this video because I don't see it.

p.s. You don't to write @lamaATS when a I was writing anything in this issue.
